examples/kuadrant/geo-loadbalanced-healthchecks-dns.yaml

```diff
@@ -0,0 +1,68 @@
+apiVersion: kuadrant.io/v1alpha1
+kind: DNSRecord
+metadata:
+  name: geo-loadbalanced-healthchecks-dns
+spec:
+  endpoints:
+    - dnsName: 14byhk-2k52h1.klb.${KUADRANT_SUB_DOMAIN}
+      recordTTL: 60
+      recordType: A
+      targets:
+        - 172.32.200.1
+    - dnsName: 24byhk-2k52h1.klb.${KUADRANT_SUB_DOMAIN}
+      recordTTL: 60
+      recordType: A
+      targets:
+        - 172.32.200.2
+    - dnsName: ${KUADRANT_SUB_DOMAIN}
+      recordTTL: 300
+      recordType: CNAME
+      targets:
+        - klb.${KUADRANT_SUB_DOMAIN}
+    - dnsName: eu.klb.${KUADRANT_SUB_DOMAIN}
+      recordTTL: 60
+      recordType: CNAME
+      setIdentifier: 14byhk-2k52h1.klb.${KUADRANT_SUB_DOMAIN}
+      targets:
+        - 14byhk-2k52h1.klb.${KUADRANT_SUB_DOMAIN}
+    - dnsName: us.klb.${KUADRANT_SUB_DOMAIN}
+      recordTTL: 60
+      recordType: CNAME
+      setIdentifier: 24byhk-2k52h1.klb.${KUADRANT_SUB_DOMAIN}
+      targets:
+        - 24byhk-2k52h1.klb.${KUADRANT_SUB_DOMAIN}
+    - dnsName: klb.${KUADRANT_SUB_DOMAIN}
+      providerSpecific:
+        - name: geo-code
+          value: GEO-EU
+      recordTTL: 300
+      recordType: CNAME
+      setIdentifier: EU
+      targets:
+        - eu.klb.${KUADRANT_SUB_DOMAIN}
+    - dnsName: klb.${KUADRANT_SUB_DOMAIN}
+      providerSpecific:
+        - name: geo-code
+          value: GEO-NA
+      recordTTL: 300
+      recordType: CNAME
+      setIdentifier: US
+      targets:
+        - us.klb.${KUADRANT_SUB_DOMAIN}
+    - dnsName: klb.${KUADRANT_SUB_DOMAIN}
+      providerSpecific:
+        - name: geo-code
+          value: '*'
+      recordTTL: 300
+      recordType: CNAME
+      setIdentifier: default
+      targets:
+        - eu.klb.${KUADRANT_SUB_DOMAIN}
+  healthCheck:
```

Comment:
so do these work with Azure then? The health checks?

Reply:
no, the health checks are only in AWS still, for now. on-cluster health checks is my next piece of work.